eclass/acct-group.eclass

```diff
@@ -156,8 +156,23 @@ acct-group_src_install() {
 acct-group_pkg_preinst() {
 	debug-print-function ${FUNCNAME} "${@}"
 
-	enewgroup ${ACCT_GROUP_ENFORCE_ID:+-F} "${ACCT_GROUP_NAME}" \
-		"${_ACCT_GROUP_ID}"
+	if egetent group "${ACCT_GROUP_NAME}" >/dev/null; then
+		elog "Group ${ACCT_GROUP_NAME} already exists"
+		return
+	fi
+
+	if [[ ${EUID} -ne 0 ]]; then
+		ewarn "Must be root to create group"
+		return
+	fi
+
+	local opts=( --system )
+	[[ -z ${ROOT} ]] || opts+=( --prefix "${ROOT}" )
```

Comment:
Please use positive logic, "if ROOT is non-empty, then ..." reads much better than "unless ROOT is empty, ...".